src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    public Figure getRandomFigure() {
+        int figureType = random.nextInt(5);

```

Comment:
Magic numbers should be replaced with named constants to improve code readability and maintainability. Consider using a constant for the number representing the types of figures.

Reply:
fix comment